src/keybindings.h

```diff
@@ -274,6 +274,7 @@ enum GeanyKeyBindingID
 	GEANY_KEYS_FORMAT_SENDTOCMD8,				/**< Keybinding. */
 	GEANY_KEYS_FORMAT_SENDTOCMD9,				/**< Keybinding. */
 	GEANY_KEYS_EDITOR_DELETELINETOBEGINNING,	/**< Keybinding. */
+	GEANY_KEYS_GOTO_ADD_POSITION,				/**< Keybinding. */
```

Comment:
There is no specified rule, so I can't see it matters either way, but I personally find underscored names more readable.